.github/workflows/cflite_pr.yml

```diff
@@ -0,0 +1,30 @@
+name: ClusterFuzzLite PR fuzzing
+on:
+  workflow_dispatch:
+  pull_request:
+    branches: [ v1.x ]
+permissions: read-all
+jobs:
+  PR:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        sanitizer: [address]
+    steps:
+    - name: Build Fuzzers (${{ matrix.sanitizer }})
+      id: build
+      uses: google/clusterfuzzlite/actions/build_fuzzers@v1
+      with:
+        sanitizer: ${{ matrix.sanitizer }}
+        language: c
+        bad-build-check: false
+    - name: Run Fuzzers (${{ matrix.sanitizer }})
+      id: run
+      uses: google/clusterfuzzlite/actions/run_fuzzers@v1
+      with:
+        github-token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Forgot to ask: what is this needed for?

Reply:
I'm not 100% sure.

I followed the template provided by the OSS-Fuzz/ClusterFuzzLite team here: https://google.github.io/clusterfuzzlite/running-clusterfuzzlite/github-actions/#pr-fuzzing

I think the secret is needed in case the fuzzers finds a crash and needs to write the payload in the GH action such that maintainers can see it. However, this is purely my intuition.

@jonathanmetzman do you know why the `secrets.GITHUB_TOKEN` is needed?

Reply:
It's for storing corpus and for downloading private repos.